homeassistant/components/velbus/__init__.py

```diff
@@ -62,7 +62,11 @@ def callback():
         load_platform(hass, 'sensor', DOMAIN,
                       discovery_info['sensor'], config)
 
+    def syn_clock(self):
```

Comment:
A parameter for the `ServiceCall` instance is missing (add `service = None`).  Did you test this locally?

Reply:
The current code worked perfectly here, all the needed messages are send out when we call the service